pokemonterminal/main.py

```diff
@@ -33,8 +33,8 @@ def daemon(time_stamp, pkmn_list):
 def slideshow(filtered, delay, changer_func):
     pid = os.fork()
     if pid > 0:
-        print(f"Starting slideshow with {len(filtered)}, pokemon " +
```

Comment:
These are not syntax errors on Python 3.6.  They are features!
* https://docs.python.org/3.6/whatsnew/3.6.html#new-features